exercise/tests_cache.py

```diff
@@ -141,10 +141,13 @@ def test_invalidation(self):
 
     def test_accumulation(self):
         self.submission2.set_points(2,2)
+        self.submission2.set_error()
         self.submission2.save()
         c = CachedContent(self.instance)
         p = CachedPoints(self.instance, self.student, c)
         entry,tree,_,_ = p.find(self.exercise)
+        # 2 submissions:
+        # ready 50, initialized 100
```

Comment:
In this test, there were two submissions: a READY one with 50 points and an INITIALIZED one with 100 points. `assertEqual` on line 150/153 would fail because the second submission is now considered the best one. This is why I decided to set its state to ERROR so it wouldn't be included in the calculation.